examples/python/Earthfile

```diff
@@ -1,12 +1,28 @@
-VERSION --global-cache 0.7
+# Common Python UDCs and Builders.
+VERSION --global-cache --use-function-keyword 0.7
+
+# Assuming the detailed setup from the second snippet is defined in relevant Earthly targets.
 
 builder:
     FROM ./../../earthly/python+python-base
 
-    COPY --dir ./src .
-    DO ./../../earthly/python+BUILDER
+    # Copy the src directory and project-specific files into /poetry in the container.
```

Comment:
Don't replace functions that already exist with inline commands.
Otherwise every python target (of which there are many across all our repos) needs to be changed to remove the use of the function.

Also, as stated above,  `build:` doesn't need to lint the python code that should be done as a `check:` target before building even occurs.

Reply:
can you check my latest commit 98485de21c7132e2bdb4c0c02ba4797d54783e5f